website/docs/r/federated_settings_identity_provider.html.markdown

```diff
@@ -58,4 +59,6 @@ Identity Provider **must** be imported before using federation_settings_id-idp_i
 $ terraform import mongodbatlas_federated_settings_identity_provider.identity_provider 6287a663c660f52b1c441c6c-0oad4fas87jL5Xnk1297
 ```
 
+**WARNING:** Starting from terraform provider version 1.16.0, to import Identity Provider, `id` will have to be used instead of `okta_idp_id`. See more [here](../guides/1.15.0-upgrade-guide.html.markdown)
```

Comment:
I believe it might be hard for users to understand what `id` is, might be better to rephrase to `24-hexadecimal digit string that identifies the IdP`

Reply:
Done